pkgs/desktops/gnome-3/core/gnome-tour/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv
+, rustPlatform
+, gettext
+, meson
+, ninja
+, fetchFromGitLab
+, pkgconfig
+, gtk3
+, glib
+, gdk-pixbuf
+, desktop-file-utils
+, appstream-glib
```

Comment:
Not used.

Reply:
Meant to put `appstream`.

Reply:
They are [using](https://gitlab.gnome.org/GNOME/gnome-tour/-/blob/5dc5ec9d47ab739ff1f2a5c1a732aa7e5483e400/meson.build#L19) `appstream-util` which comes from `appstream-glib`.

Reply:
Oooh, I forgot that `appstream-util` was in `dev`. So I assumed it had to have just been `appstream`. Will fix.